src/MSALWrapper.Test/AuthFlow/AuthFlowExecutorTest.cs

```diff
@@ -96,37 +97,50 @@ public void ConstructorWith_Valid_Arguments()
         public async Task SingleAuthFlow_Returns_TokenResult()
         {
             var authFlow1 = new Mock<IAuthFlow>(MockBehavior.Strict);
-            var authFlowResult = new AuthFlowResult(this.tokenResult, null, string.Empty);
+            var authFlowName = "authFlowName1";
+            var authFlowResult = new AuthFlowResult(this.tokenResult, null, authFlowName);
+            var authFlowResultList = new List<AuthFlowResult>();
+            authFlowResultList.Add(authFlowResult);
             authFlow1.Setup(p => p.GetTokenAsync()).ReturnsAsync(authFlowResult);
 
             // Act
-            var authFlow = this.Subject(new[] { authFlow1.Object });
-            var result = await authFlow.GetTokenAsync();
+            var authFlowExecutor = this.Subject(new[] { authFlow1.Object });
+            var result = await authFlowExecutor.GetTokenAsync();
+            var resultList = result.ToList();

```

Comment:
Just a thought, but maybe... `results` instead of `resultList`? The compiler knows what the type is.

Reply:
Just wanted to differentiate the Enumerable result and List result so used `resultList`.

Reply:
Could clean up more in the future - you could tack on the `.ToList()` to the `GetTokenAsync()` so you only have 1 `results` variable and it is a List.

Reply:
Or an Array